src/frontend/src/catalog/system_catalog/information_schema/columns.rs

```diff
@@ -33,12 +35,28 @@ pub const INFORMATION_SCHEMA_COLUMNS: BuiltinTable = BuiltinTable {
         (DataType::Varchar, "column_name"),
         (DataType::Varchar, "column_default"),
         (DataType::Int32, "character_maximum_length"),
-        (DataType::Int32, "ordinal_position"),
+        (DataType::Int16, "ordinal_position"),
         (DataType::Varchar, "is_nullable"),
         (DataType::Varchar, "collation_name"),
         (DataType::Varchar, "udt_schema"),
         (DataType::Varchar, "data_type"),
         (DataType::Varchar, "udt_name"),
     ],
-    pk: &[],
-};
+    sql: "SELECT CURRENT_DATABASE() AS table_catalog, \
+                s.name AS table_schema, \
+                r.name AS table_name, \
+                c.name AS column_name, \
+                NULL AS column_default, \
+                NULL::integer AS character_maximum_length, \
+                c.position AS ordinal_position, \
+                'YES' AS is_nullable, \
+                NULL AS collation_name, \
+                'pg_catalog' AS udt_schema, \
+                c.data_type AS data_type, \
+                c.udt_type AS udt_name \
+            FROM rw_catalog.rw_columns c \
+            LEFT JOIN rw_catalog.rw_relations r ON c.id = r.id \
```

Comment:
`c.relation_id = r.id`

Before fixing the definition here, how about writing a test that iterates over all `BuiltinView` to make sure their `sql` binds without problem?

Reply:
Okay, I will add one.